package.json

```diff
@@ -35,7 +35,6 @@
     "column-resizer": "^1.3.5",
     "downshift": "^6.1.2",
     "gh-pages": "^2.0.0",
-    "i": "^0.3.6",
```

Comment:
This library does the same thing as pluralize but is less popular and I'm 99% sure we don't use it. Maybe it was accidentally installed?

Reply:
Seems likely